desktop/package/package.gradle

```diff
@@ -263,26 +263,41 @@ task packageInstallers {
                         " --app-version ${appVersion}" +
                         " --copyright \"${appCopyright}\"" +
                         " --vendor ${appNameAndVendor}" +
-                        " --temp \"${jpackageTempDir}\"" +
-
-                        // Options for creating the application image
-                        " --input ${fatJarFolderPath}" +
-
-                        // Options for creating the application launcher
-                        " --main-jar ${mainJarName}" +
-                        " --main-class haveno.desktop.app.HavenoAppMain" +
-                        " --java-options -Xss1280k" +
-                        " --java-options -XX:MaxRAM=4g" +
-                        " --java-options --add-opens=javafx.controls/com.sun.javafx.scene.control.behavior=ALL-UNNAMED" +
-                        " --java-options --add-opens=javafx.controls/com.sun.javafx.scene.control=ALL-UNNAMED" +
-                        " --java-options --add-opens=java.base/java.lang.reflect=ALL-UNNAMED" +
-                        " --java-options --add-opens=javafx.graphics/com.sun.javafx.scene=ALL-UNNAMED" +
-                        " --java-options -Djava.net.preferIPv4Stack=true" +
-                        " --arguments --baseCurrencyNetwork=XMR_STAGENET"
+                        " --temp \"${jpackageTempDir}\""
+        )
+
+        // stores temporary app-image directory
+        File jpackageAppImageContainer = new File(tempRootDir, "jpackage-app-image")
+        File jpackageAppImage = new File(jpackageAppImageContainer, "Haveno")
+        jpackageAppImageContainer.mkdirs()
+
+        // package generic app-image
+        executeCmd(jPackageFilePath + commonOpts +
+                // Options for creating the application launcher
+                " --main-jar ${mainJarName}" +
```

Comment:
Getting a build error on macOS.

I see that these options will not be part of `commonOpts` for the `executeCmd` commands below, in case that could have anything to do with it.

Reply:
> Getting a CI failure for macOS.

The CI build error is `mv: rename desktop/build/temp-*/binaries/Haveno-*.dmg to /Users/runner/work/haveno/haveno/release/Haveno-1.0.11-mac-installer.dmg: No such file or directory`, so looks like the macOS installer isn't built for some reason.

Reply:
Checking it out now